app/services/request_approval_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RequestApprovalService < BaseService
+  def call(user)
+    return true if user.id.nil?
+
+    if Setting.require_approval
+      User.admins.each do |admin|
```

Comment:
If you look at the mailer for reports, usually the recipients (admins) are queried from within the mailer, rather than the other way around like here.